docs/src/test/java/docs/http/javadsl/server/directives/CachingDirectivesExamplesTest.java

```diff
@@ -52,26 +55,29 @@ public Uri apply(RequestContext in, boolean isCheck) {
     };
 
     final AtomicInteger count = new AtomicInteger(0);
-    final Route route = cache(routeCache(cachingSettings), simpleKeyer, () ->
-      extractUri(uri ->
-        complete(String.format("Request for %s @ count %d", uri, count.incrementAndGet()))
+    final Cache<Uri, RouteResult> myCache = routeCache(cachingSettings);
```

Comment:
Maybe even add a comment about creating it outside the routes to have a shared single cache instance?